SQL/Archive/2018-05-28_publication_schema.sql

```diff
@@ -0,0 +1,160 @@
+SET FOREIGN_KEY_CHECKS=0;
+-- Publication Status
+DROP TABLE IF EXISTS publication_status;
+CREATE TABLE `publication_status` (
+  `PublicationStatusID` int(2) unsigned NOT NULL,
+  `Label` varchar(255) NOT NULL,
+  PRIMARY KEY(`PublicationStatusID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+DELETE FROM publication_status;
+INSERT INTO publication_status VALUES (1, 'Pending');
+INSERT INTO publication_status VALUES (2, 'Approved');
```

Comment:
`PublicationStatusID` should be auto_increment'ing and its value should be omitted in the insert statement. 
(Except for the first value that need to be 1 for the default value in the referencing table `publication` )